src/content/blog/SwitchPackageToProjRef.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Seemless switching between Package and Project references
+date: 2023-11-11
+author: "dawe"
+slug: "2023/11/11"
+---
+
+## Switching seemlessly between Package and Project references - 2023-11-11
+
+##### by dawe
+
+While doing open source development on a code base with package dependencies which are crucial to your work, you easily run into a situation where you want to adjust the package code to your needs. Reasons could be a missing feature, a bug, or just a different opinion on how things should be done.  
+Depending on the nature of your work, you might want to do the changes in the upstream code and consume them right away in the consuming project to finish your original work. This has also the benefit that you see the impact of your changes right away and can adjust them if needed.
+
+We had such a situation recently while working on the [Ionide-Analyzers](https://github.com/ionide/ionide-analyzers). There was a bug in the [FSharp.Analyzers.SDK](https://github.com/ionide/FSharp.Analyzers.SDK) blocking [Florian](https://github.com/nojaf) from finishing his analyzer.
```

Comment:
Please link to https://www.youtube.com/live/H7w4NfF6xsg?si=kuCL3CcevR5Bsqgg&t=1406 instead of my GitHub profile